.github/workflows/build.yml

```
@@ -27,4 +27,9 @@ jobs:
         run: bun run build
 
       - name: Test
-        run: bun test
+        run: bun test --coverage --coverage-reporter=lcov
+
+      - name: Upload coverage reports to Codecov
+          uses: codecov/codecov-action@v5
```

actionlint (1.7.10)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)